### ISSUE TITLE
can't select multiple files at the same time

### ISSUE THREAD

**Issue by letroll** [iPaulPro/aFileChooser#2]
can you add the ability to select multiple files at the same time, and if possible at the same time a box "select all / deselect all" ?

**Comment by iPaulPro** [comment]
Hey, this is a bit out of scope, but something I have considered. I am, mostly, trying to follow the Android APIs and docs, using `ACTION_GET_CONTENT` with `CATEGORY_OPENABLE` to return a URI: http://developer.android.com/reference/android/content/Intent.html#ACTION_GET_CONTENT

**Comment by paulasiimwe** [comment]
Much as it is abit out of scope I believe it's something not so hard for you. Simply implement checkboxes alongside each listed file and when the user clicks ok, return a uri array. Please Paul! help us please please please!!!

**Comment by LOG-TAG** [comment]
I'm want share the Stackoverflow discussion  http://stackoverflow.com/questions/22095441/android-multiple-file-selector-chooser-dialog

**Comment by paulasiimwe** [comment]
Yes I am the one who posted it on stackoverflow. The proposed solution is something im hoping someone will integrate into this great File Selector.

**Comment by paulasiimwe** [comment]
Simple Solution here https://github.com/tapaulo/Parian-Android-Multiple-file-Selector-Dialog
